docs/developers_guide/organization/steps.md

```diff
@@ -357,6 +357,37 @@ attributes.  Another reason might be to set these attributes using an algorithm
 When overriding `constrain_resources`, it is important to also call the base
 class' version of the method with `super().constrain_resources()`.
 
+The names of the resources are related to the
+[Slurm](https://slurm.schedmd.com/srun.html) naming conventions:
+
+``ntasks``
+
+: The target number of MPI tasks that a step will use if the resources are available.
+
+``min_tasks``
+
+: The minimum number of MPI tasks for a step.  If too few resources are
+available, the step will not run.
+
+``cpus_per_task``
+
+: If ``ntasks > 1``, this is typically a number of threads used by each
+MPI task (e.g. with OpenMP threading).  If ``ntasks == 1``, this may
+be the number of target cores used in on-node parallelism like python or
```

Comment:
One more thing if you don't mind me being picky.  These read a little more easily if we indent all the lines after the first one by 2 spaces (see the examples further up in the document).

Reply:
This is just for code readability, right, not something that will be in the final document? Just making sure I'm not supposed to be seeing a change on the build side when I indent this, because it looks the same.

Reply:
@xylar I made the change, let me know if it looks wrong!

Reply:
Yes, exactly.  It's just to make the documents a little easier to read before they get rendered into html, it has no effect on the resulting documentation in the end.  Looks great, thanks again!